QuickOSM/definitions/format.py

```diff
@@ -0,0 +1,17 @@
+"""Definitions for output formats."""
+
+from enum import Enum, unique
+
+__copyright__ = 'Copyright 2019, 3Liz'
+__license__ = 'GPL version 3'
+__email__ = 'info@3liz.org'
+
+
+@unique
+class Format(Enum):
+    """ Name of output format."""
+    GeoJSON = 'geojson'
+    GPKG = 'gpkg'
+    Shapefile = 'shp'
+    Shapefile_name = 'ESRI Shapefile'
```

Comment:
A enum is designed to store **one** time. Here, you have 2 formats possible for the shapefile.

If you want to store extra properties, you "NamedTuple" as well.

"label", "driver_name" etc